services/configuration/src/config_api.rs

```diff
@@ -70,54 +72,52 @@ pub struct PublicConfigApi {
 }
 
 impl PublicConfigApi {
-    fn get_actual_config(&self) -> Result<ApiResponseConfigHashInfo, ApiError> {
-
-        let actual_cfg = Schema::new(&self.blockchain.snapshot()).actual_configuration();
-        let res = ApiResponseConfigHashInfo {
-            hash: actual_cfg.hash(),
-            config: actual_cfg,
-        };
-        Ok(res)
+    fn get_config_with_proofs(&self, config: StoredConfiguration) -> ApiResponseConfigHashInfo {
+        let propose = ConfigurationSchema::new(&self.blockchain.snapshot())
```

Comment:
Nitpick: ref'ing a snapshot seems superfluous here.

Reply:
I want as many such "nitpicks" as possible: they makes the code better. :+1: